docs/source/install.rst

```diff
@@ -0,0 +1,35 @@
+Install Dask-Gateway
+====================
+
+As mentioned in :doc:`the architecture overview <overview>`, dask-gateway is split into severl parts.  The pieces can easily be install with two packages (pip or conda) or can be installed from source.  Typically, we don't expect users to install from source unless adding new features and/or debugging.
+
+
+Conda
+-----
+
+To install the latest version of dask-gateway from the
+`conda-forge <https://conda-forge.github.io/>`_ repository using
+`conda <https://www.anaconda.com/downloads>`_::
+
+    conda install dask-gateway dask-gateway-server -c conda-forge
```

Comment:
probably worth mentioning somewhere around here that there are a number of alternatives that you will likely need depending on your cluster backend